configs/template_config.yaml

```diff
@@ -122,6 +124,8 @@ rand_seed: 42
 precision: fp16
 # Save Top K (int): Number of best models to save; default saves the best single checkpoint. Set to -1 to save all
 save_top_k: 3
+# Every N hours (int or float): Checkpoint every n hours; overrides every_n_train_steps if not set to default (None or 0). 
+every_n_hours: ~
 # Every N Train Steps (int): Checkpoint every n train steps; Note: PyTorch Lightning defines 'steps' as 
 # optimizer steps, not batch forward passes, distinct when accumulate_grad_batches > 1.
 every_n_train_steps: 10000
```

Comment:
Do we also want to add a reference to the `every_n_hours` parameter above, similar as we did on line #127?

Reply:
Oh, like "# Overridden by `every_n_hours` parameter if not `0` or `None`"?

Reply:
Addressed in https://github.com/DRAGNLabs/301r_retnet/commit/1e597ed10da4483b55c2d3e0fb552192f755d92a.